File: examples/corpus/ts/01_pow_mod.after.ts
function power(base: number, exponent: number): number {
  return base ** exponent;
}

function square(x: number): number {
  return x * x;
}

const MOD = 1_000_000_007;

function sumMod(values: number[]): number {
  return values.reduce((a, b) => a + b, 0) % MOD;
}

File: examples/corpus/go/01_pow_mod.after.go
package main

import "math"

func power(base, exp int) int {
	return int(math.Pow(float64(base), float64(exp)))
}

func square(x int) int {
	return x * x
}

const MOD = 1_000_000_007

func sumMod(values []int) int {
	total := 0
	for _, v := range values {
		total += v
	}
	return total % MOD
}
